infrastructure/metadata/infrastructure/assumptions/html-elements.html.ini

```diff
@@ -0,0 +1,6 @@
+[html-elements.html]
+  [Compare CSS div definitions (only valid if pre-reqs pass)]
+    expected:
+      if product == "chrome": PASS
```

Comment:
Are you running this on master? On master, after #9674, only Firefox should pass!